force_wfmanager/wfmanager_task.py

```diff
@@ -193,6 +203,15 @@ def open_workflow(self):
             else:
                 self.current_file = dialog.path
 
+    def open_about(self):
+        information(
+            None,
+            "Workflow manager: UI application for Business Decision System\n\n"
+            "It is part of the FORCE project\n"
+            "This software is released under the BSD license",
+            "About WorflowManager"
```

Comment:
Maybe it is unclear, but "About WorkflowManager" is the name of the dialog. Do you me to remove this line ? The default title is "Information"